tests/unit/workflows/test_workflows_tasks_arxiv.py

```diff
@@ -181,6 +181,98 @@ def test_populate_arxiv_document_logs_on_pdf_not_existing():
         assert expected == result
 
 
+def test_populate_arxiv_document_documents_not_existing():
+    response500 = {'content': '', 'status_code': 500}
+    response200 = {
+        'content': pkg_resources.resource_string(
+            __name__, os.path.join('fixtures', '1707.02785.html')),
+        'status_code': 200,
+    }
+    with requests_mock.Mocker() as requests_mocker:
+        requests_mocker.get(
+            'http://export.arxiv.org/pdf/1707.02785',
+            (response200,),
+        )
+        requests_mocker.get(
+            'http://arxiv.org/pdf/1707.02785',
+            (response500,),
+        )
+        schema = load_schema('hep')
+        subschema = schema['properties']['arxiv_eprints']
+
+        data = {
+            'arxiv_eprints': [
+                {
+                    'categories': [
+                        'cs.CV',
```

Comment:
I think that in order to test the code that you wrote you need to add `documents` key, which should be emptied [here](https://github.com/inspirehep/inspire-next/blob/87326f7b0417a14dced1bc59f43e17dc29ca2ff3/inspirehep/modules/workflows/tasks/actions.py#L401-L403) cause we are deleting documents what could result in something like `documents: []` and then we assign workflow data to the builder. Same for other tests